test/e2e/tests/test_db_cluster.py

```diff
@@ -51,60 +52,77 @@
 
 MODIFY_WAIT_AFTER_SECONDS = 20
 
+# MUP == Master user password...
+MUP_NS = "default"
+MUP_SEC_NAME = "dbclustersecrets"
+MUP_SEC_KEY = "master_user_password"
+MUP_SEC_VAL = "secretpass123456"
+
+
+@pytest.fixture
+def aurora_mysql_cluster(k8s_secret):
+    db_cluster_id = random_suffix_name("my-aurora-mysql", 20)
+    db_name = "mydb"
+    secret = k8s_secret(
+        MUP_NS,
+        MUP_SEC_NAME,
+        MUP_SEC_KEY,
+        MUP_SEC_VAL,
+    )
+
+    replacements = REPLACEMENT_VALUES.copy()
+    replacements['COPY_TAGS_TO_SNAPSHOT'] = "False"
+    replacements["DB_CLUSTER_ID"] = db_cluster_id
+    replacements["DB_NAME"] = db_name
+    replacements["MASTER_USER_PASS_SECRET_NAMESPACE"] = secret.ns
+    replacements["MASTER_USER_PASS_SECRET_NAME"] = secret.name
+    replacements["MASTER_USER_PASS_SECRET_KEY"] = secret.key
+
+    resource_data = load_rds_resource(
+        "db_cluster_mysql_serverless",
+        additional_replacements=replacements,
+    )
+
+    ref = k8s.CustomResourceReference(
+        CRD_GROUP, CRD_VERSION, RESOURCE_PLURAL,
+        db_cluster_id, namespace="default",
+    )
+    k8s.create_custom_resource(ref, resource_data)
+    cr = k8s.wait_resource_consumed_by_controller(ref)
+
+    assert cr is not None
+    assert 'status' in cr
+    assert 'status' in cr['status']
+    assert cr['status']['status'] == 'creating'
+    condition.assert_not_synced(ref)
+
+    db_cluster.wait_until(
+        db_cluster_id,
+        db_cluster.status_matches('available'),
+    )
+
+    assert cr is not None
+    assert k8s.get_resource_exists(ref)
```

Comment:
This is already asserted (or assumed) above

Reply:
removed.